src/tests/function/random_uuid.rs

```diff
@@ -0,0 +1,26 @@
+use crate::*;
+
+test_case!(random_uuid, async move {
+    let test_cases = vec![
+        ("CREATE TABLE SingleItem (id UUID)", Ok(Payload::Create)),
+        (
+            r#"INSERT INTO SingleItem VALUES (random_uuid())"#,
+            Ok(Payload::Insert(1)),
+        ),
+        (
+            "SELECT random_uuid(0) as pi FROM SingleItem",
+            Err(TranslateError::FunctionArgsLengthNotMatching {
+                name: "RANDOM_UUID".to_owned(),
+                expected: 0,
+                found: 1,
+            }
+            .into()),
+        ),
```

Comment:
What do you think about adding test code for uuid pattern matching?

Reply:
I will add test code for pattern matching.
@panarch 
Are there any macro in gluesql can use for pattern matching? Or should I create a new one?

ex) 
```
(
    "SELECT RANDOM_UUID() as UUID FROM SingleItem",
    Ok(type_match!(
        UUID
        Value::UUID;
    )),
),
```